Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Description: ensure external SPM(F) is found when FORTRAN interface is enabled
------------------------------------------------------------------------------
diff -ruN pastix-v6.2.2/CMakeLists.txt pastix-v6.2.2_p1/CMakeLists.txt
--- pastix-v6.2.2/CMakeLists.txt	2022-09-08 13:26:04.000000000 +0100
+++ pastix-v6.2.2_p1/CMakeLists.txt	2022-09-09 16:59:42.000000000 +0100
@@ -435,7 +435,8 @@
 
 if ( PASTIX_WITH_EXTERNAL_SPM )
   if ( PASTIX_WITH_FORTRAN )
-    find_package( SPM QUIET COMPONENTS FORTRAN )
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(SPM spm spmf REQUIRED)
   else()
     find_package( SPM QUIET )
   endif()
